mosip/on-prem/README.md

```diff
@@ -43,7 +43,21 @@ ansible-playbook -i hosts.ini ports.yaml
 ```
 ansible-playbook -i hosts.ini swap.yaml
 ```
+## Localhost Mapping
+Below contains some of the information about localhost mapping in brief:
 
+we faced an issue with nodal instances where we were getting issues as localhost mapping was not present in hosts file of the system.
+Reason: This is because of localhost mapping i,e ```127.0.0.1    localhost``` might not being present within ```/etc/hosts``` file of the system.
```

Comment:
Issue: Localhost mapping i.e. ```127.0.0.1    localhost``` might not being present within ```/etc/hosts``` file of the system.